spec/integration/cli_view_spec.rb

```diff
@@ -24,5 +24,37 @@
       expect(File.exist?(File.join(output_dir, "libraries", "awesomelib.html"))).to be(true)
       expect(File.exist?(File.join(output_dir, "libraries", "newlib.html"))).to be(true)
     end
+
+    context "inqlude-all.json" do
+      it "generates view" do
+        output_dir = given_directory
+        result = run_command(args: ["view", "--output-dir=#{output_dir}"])
```

Comment:
You are right. This is a limitation of given_filesystem. I suppose it's best to just use `before(:each)` for now.